xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/ComputedFieldClass.java

```diff
@@ -108,25 +149,7 @@ public BaseProperty newProperty()
     public void displayView(StringBuffer buffer, String name, String prefix, BaseCollection object,
         XWikiContext context)
     {
-        String script = getScript();
-
-        try {
-            ScriptContext scontext = Utils.getComponent(ScriptContextManager.class).getCurrentScriptContext();
-            scontext.setAttribute("name", name, ScriptContext.ENGINE_SCOPE);
-            scontext.setAttribute("prefix", prefix, ScriptContext.ENGINE_SCOPE);
-            scontext.setAttribute("object", new com.xpn.xwiki.api.Object((BaseObject) object, context),
-                ScriptContext.ENGINE_SCOPE);
-
-            XWikiDocument classDocument = object.getXClass(context).getOwnerDocument();
-
-            String result = renderContentInContext(script, classDocument.getSyntax().toIdString(),
-                classDocument.getAuthorReference(), classDocument.getDocumentReference(), context);
-
-            buffer.append(result);
-        } catch (Exception e) {
-            // TODO: append a rendering style complete error instead
-            buffer.append(e.getMessage());
-        }
+        buffer.append(getComputedValue(name, prefix, object, context));
```

Comment:
Either call ``getComputedValue(buffer, ...)`` here or remove the second signature of ``getComputedValue`` since I don't see it used anywhere.

Reply:
Indeed, I removed the signature altogether since it was not used.